lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -513,5 +513,221 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_CHARGE_ITEM_CREATE ||= {
+      'resourceType': 'Parameters',
+      'parameter': [
+        {
+          'name': 'chargeItem',
+          'resource': {
+            'resourceType': 'ChargeItem',
+            'status': 'unknown',
+            'subject': {
+              'reference': 'Patient/178594204'
+            },
+            'context': {
+              'reference': 'Encounter/53066085'
+            },
+            'occurrenceDateTime': '2021-08-02T21:53:34Z',
+            'quantity': {
+              'value': 1
+            },
+            'code': {
+              'coding': [
+                {
+                  'extension': [
+                    {
+                      'valueCoding': {
+                        'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                        'code': '667687'
+                      },
+                      'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule'
+                    }
+                  ],
+                  'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED',
+                  'code': '01012021'
+                }
+              ]
+            },
+            'performer': [
+              {
+                'function': {
+                  'coding': [
+                    {
+                      'system': 'http://terminology.hl7.org/CodeSystem/v2-0912',
+                      'code': 'OP',
+                      'display': 'Ordering Provider'
```

Comment:
Do we actually support/validate the display and the text for the performer function on a create?